Taskfile.yaml

```diff
@@ -1,36 +1,76 @@
 version: "3"
 
 includes:
-  vue:
-    taskfile: ./internal/blockchain/vue/Taskfile.yaml
-    dir: internal/blockchain/vue
+  web:dashboard:
+    taskfile: ./web/dashboard/Taskfile.yaml
+    dir: web/dashboard
+  web:wallet:
+    taskfile: ./web/wallet/Taskfile.yaml
+    dir: web/wallet
 
 tasks:
-  build-cli:
-    dir: cmd/highway-cli
+  bind:
     cmds:
-      - echo "Building Sonr Highway CLI"
-      - go build -o highway-cli main.go
+      - echo "Setting up Gomobile..."
+      - go install golang.org/x/mobile/cmd/gomobile@latest
+      - go mod download golang.org/x/mobile
+      - gomobile init
+      - task: bind:ios
+      - task: bind:android
 
-  build-swagger:
-    dir: pkg
+  bind:android:
     cmds:
-      - echo "Building swagger api docs for highway"
-      - swag i -g pkg/highway/server.go --output docs/highway
+      - echo "Building Sonr Motor Node (Android)"
+      - gomobile bind -ldflags='-s -w' -target=android -o .../dist/io.sonr.motor.aar
+
+  bind:ios:
+    cmds:
+      - echo "Building Sonr Motor Node (iOS)"
+      - gomobile bind -ldflags='-s -w' -bundleid=io.sonr.motor -target=ios -o .../dist/SonrMotor.xcframework
+
+  build:
+    cmds:
+      - task: build:buf
+      - task: build:cli
+      - task: build:swagger
+      - task: build:chain
+
+  build:buf:
+    dir: pkg/proto
+    cmds:
+      - echo "Generating and Pushing Protobufs"
+      - buf mod update
+      - buf build
+      - buf push
 
-  chain-build:
+  build:chain:
     summary: Build the project for production without release mode
     cmds:
       - starport chain build
     silent: true
 
-  chain-serve:
+  build:cli:
+    dir: cmd/highway-cli
+    cmds:
+      - echo "Building Sonr Highway CLI"
+      - go build -o highway-cli main.go
+
+  build:swagger:
+    cmds:
+      - echo "Building swagger api docs for highway"
+      - swag i -g pkg/highway/server.go --output docs/highway
+      - echo "Building Open api docs for Blockchain"
+      - starport generate openapi
+      - starport generate proto-go
+      - go mod tidy
+
+  chain:serve:
```

Comment:
nit: You have `build:chain` but `chain:serve`. I think these should be made consistent

Reply:
okay serve:chain